experiments/mcp-eval/tasks/create-yaml/report.txt

```diff
@@ -0,0 +1,44 @@
+Evaluation Report
+====================
+--- Summary for MCP Disabled ---
+Total Tests Run: 13
+Tests Passed: 10
+Tests Failed: 3
+Pass Rate: 76.92%
+Average Latency: 41274.06 ms
+Max Latency: 87374.02 ms
+Total LLM Requests: 0
+
+--- Summary for MCP Enabled ---
+Total Tests Run: 13
+Tests Passed: 11
+Tests Failed: 2
+Pass Rate: 84.62%
+Average Latency: 60162.33 ms
+Max Latency: 104002.52 ms
+Total LLM Requests: 0
+
+======= Comparison Report: MCP vs. No MCP =======
+
+                   test_name  passed_mcp  passed_no_mcp  latency_ms_mcp  latency_ms_no_mcp  latency_diff  llm_requests_mcp  llm_requests_no_mcp  llm_requests_diff
+
+
+======= Comparison Report: MCP vs. No MCP =======
+
+                   test_name  passed_mcp  passed_no_mcp  latency_ms_mcp  latency_ms_no_mcp  latency_diff  llm_requests_mcp  llm_requests_no_mcp  llm_requests_diff
+0       storagebucket-1-ubla        True          False    82686.204672       47794.946671  34891.258001                 0                    0                  0
```

Comment:
OK, so this is the one!  For further improvements, it is usually good to look at why passed_no_mcp failed, it can be instruction in improving the MCP (and sometimes the non-MCP prompt)